test/unit/authentication/authentication_test.js

```diff
@@ -86,41 +89,35 @@ describe('external browser authentication', function ()
         return;
       }
     });
-    mock('httpclient', {
-      post: async function (url, body, header)
-      {
-        var data =
-        {
-          data: {
-            data:
-            {
-              ssoUrl: mockSsoURL,
-              proofKey: mockProofKey
-            }
+    mock('ssoUrlProvider', {
```

Comment:
cannot we still mock at the axios level?

Reply:
fixed